test/jasmine/tests/gl3d_plot_interact_test.js

```diff
@@ -21,6 +21,133 @@ function countCanvases() {
     return d3.selectAll('canvas').size();
 }
 
+describe('Test gl3d before/after plot', function() {
+    var gd;
+
+    var mock = require('@mocks/gl3d_marker-arrays.json');
+
+    beforeEach(function() {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 4000;
+    });
+
+    afterEach(function() {
+        Plotly.purge(gd);
+        destroyGraphDiv();
+    });
+
+    it('@noCI @gl should not rotate camera on the very first click before scene is complete and then should rotate', function(done) {
```

Comment:
Can you tell me why you added `@noCi`? Is this test always failing on CircleCi?

Reply:
Good point.
The test failed on CI once added: https://circleci.com/gh/plotly/plotly.js/27617.
Am running it again to see it may pass.

Reply:
No success there the second run: https://circleci.com/gh/plotly/plotly.js/27814.
So `@noCI` flag (as it provided) may be required.